custom_dc/custom_data.md

```diff
@@ -116,11 +167,209 @@ geoId/06,2021,555,666
+  - `name`: A human-friendly readable name that will be shown throughout the UI.
+  - `description`: A more detailed name that will be shown in the Statistical Variable Explorer.
+  - `searchDescriptions`: This is a comma-separated list of natural-language text descriptions of the variable; these descriptions will be used to generate embeddings for the NL query interface.
+  - `group`: This will display the variables as a group in the Statistical Variable Explorer, using the name you provide as heading. 
+
+The other fields are explained in the [Data config file specification reference](#json-ref).
+
+## Prepare your data using explicit schema
+
+In this section, we will walk you through a concrete example of how to go about setting up your CSV, MCF and JSON files.
+
+### Write the MCF file {#mcf}
+
+Nodes in the Data Commons knowledge graph are defined in Metadata Content Format (MCF). For custom Data Commons using explicit schema, you must define your statistical variables using MCF. Here's an example of defining the same statistical variables in the WHO data in MCF:
+
+```
+Node: dcid:Adult_curr_cig_smokers
+typeOf: dcid:StatisticalVariable
+name: "Prevalence of current cigarette smoking among adults (%)"
+populationType: dcid:Person
+measuredProperty: dcid:percent
+
+Node: dcid:Adult_curr_cig_smokers_female
+typeOf: dcid:StatisticalVariable
+name: "Prevalence of current cigarette smoking among adults (%) [Female]"
+populationType: dcid:Person
+measuredProperty: dcid:percent
+gender: dcid:Female
+
+Node: dcid:Adult_curr_cig_smokers_male
+typeOf: dcid:StatisticalVariable
+name: "Prevalence of current cigarette smoking among adults (%) [Male]"
+populationType: dcid:Person
+measuredProperty: dcid:percent
+gender: dcid:Male
+```
+The following fields are always required:
+- `Node`: This is the DCID of the entity you are defining. 
+- `typeOf`: In the case of statistical variable, this is always `dcid:StatisticalVariable`.
+- `name`: This is the descriptive name of the variable, that is displayed in the Statistical Variable Explorer and various other places in the UI.
+- `populationType`: This is the type of thing being measured, and its value must be an existing `Class` type. It is mainly used to classify variables into categories that appear in the Statistical Variable Explorer. In this example it is `dcid:Person`. For a full list of supported classes, you will have to send an API request, as described in [Get a list of all existing statistical variables](/api/rest/v2/node.html#liststatvars).
+- `dcid:measuredProperty`: This is a property of the thing being measured. It must be a `domainIncludes` property of the `populationType` you have specified. In this example, it is the `percent` of persons being measured. You can see the set of `domainIncludes` properties for a given `populationType`, using either of the following methods:
```

Comment:
Instead of 'It must be a `domainIncludes` property of the `populationType` you have specified.', since we specify the domainIncludes in the node for property we could say:
' The  `populationType` should be set as a `domainIncludes` value for the `measuredProperty`.'

Reply:
Hmm, conceptually though, I think people think first of the population and then the property they want to measure. It's weird to talk about a property to measure and then the population, don't you think?